### PR TITLE
Add comment and correspondence writing support to RT. The reading is already available via @rt.history (closes #18)

### LINKED ISSUE #18: Support comments / correspondence on tickets
It'd be nice to be able to add comments / correspondence via roust.

### PR DESCRIPTION
**Pull request by Hobbsee**

### CODE REVIEW

**Comment by Hobbsee** [comment]
@auxesis could you please review this?

File: lib/roust/ticket.rb [hunk 1]
```diff
@@ -82,6 +82,35 @@ def ticket_update(id, attrs)
         end
     end
 
+    def ticket_comment(id, attrs)
+      attrs['Text'].gsub!(/\n/, "\n ") if attrs['Text'] # insert a space on continuation lines.
+      content = compose_content('ticket', id, attrs)
+
+      unless content.match(/Action: (comment|correspond)/i)
```
Comment (johnf) [inline lib/roust/ticket.rb]: Would this be better tested on ```attrs``` i.e. before content compose. could then be
```
unless ['comment', 'correspond'].include(attrs['Action'])
```

File: spec/roust/ticket_spec.rb [hunk 2]
```diff
@@ -60,6 +60,16 @@
                   :body    => mocks_path.join('ticket-100-update.txt').read,
                   :headers => {})
 
+    stub_request(:post, "http://rt.example.org/REST/1.0/ticket/100/comment")
+      .with { |request|
+        query = WebMock::Util::QueryMapper.query_to_values(request.body)
+        require 'pry'
```
Comment (johnf) [inline spec/roust/ticket_spec.rb]: should this still be here?

**Comment by johnf** [comment]
A couple of small comments. Otherwise merge it!

File: spec/roust/ticket_spec.rb (hunk 2 above)
Comment (Hobbsee) [inline spec/roust/ticket_spec.rb]: Weirdly, yes.  It's almost identical to the stanza above it.

File: lib/roust/ticket.rb (hunk 1 above)
Comment (Hobbsee) [inline lib/roust/ticket.rb]: The reason I haven't done this is that compose_content normalises the case and type of the keys.  Doing the Action check above would mean that the other fields could use symbols, and case-insensitive keys, but not Action.  That doesn't seem like a great user experience, and this seemed like a better solution.

**Pull request merged**